solr/core/src/java/org/apache/solr/util/IndexOutputOutputStream.java

```diff
@@ -20,11 +20,12 @@
 import java.io.OutputStream;
 import org.apache.lucene.store.IndexOutput;
 
-public class PropertiesOutputStream extends OutputStream {
+/** Wraps an {@link IndexOutput} to expose it as an {@link OutputStream}. */
```

Comment:
thank you!   Helps me understand what it's for (and why the double "Output" in the name!)...